src/js/components/complete.jsx

```diff
@@ -18,7 +18,7 @@ class Complete extends Component {
           This meeting is now over. There is nothing left to see here. Move along.
         </div>
         <div className='oops'>
-          Didn't mean to end the meeting? <a href='#' onClick={this.goBack}>Go Back</a>
+          'Didn\'t mean to end the meeting?' <a href='#' onClick={this.goBack}>Go Back</a>
```

Comment:
Dude, this really, really doesn't need quotes around it. Like for real. Totally not screwing with you =) If its throwing off the highlighting in your editor you should download a JSX syntax highlight plugin